gym/wrappers/record_episode_statistics.py

```diff
@@ -9,33 +10,34 @@
 class RecordEpisodeStatistics(gym.Wrapper):
     """This wrapper will keep track of cumulative rewards and episode lengths.
 
-    At the end of an episode, the statistics of the episode will be added to `info`. After the completion
-    of an episode, `info` will look like this:
+    Example:
+        At the end of an episode, the statistics of the episode will be added to ``info``. After the completion
+        of an episode, ``info`` will look like this::
```

Comment:
I would remove `Example:` and dedent this region. The code should still be rendered properly. I just think that the text is valid in general and not part of a single example

Reply:
Agreed, updated